sys/net/application_layer/nanocoap/nanocoap.c

```diff
@@ -176,6 +176,18 @@ uint8_t *coap_find_option(const coap_pkt_t *pkt, unsigned opt_num)
     return NULL;
 }
 
+/*
+ * Parse option attributes
+ *
+ * pkt[in]           coap_pkt_t for buffer
+ * pkt_pos[in,out]   in: first byte of option in buffer
```

Comment:
how can this be "out"? It is a pointer, not pointer to pointer.

Reply:
Thanks for correcting the sloppiness.